packages/vuetify/src/components/VNavigationDrawer/VNavigationDrawer.ts

```diff
@@ -442,7 +439,7 @@ export default baseMixins.extend({
         !this.$el
       ) return 0
 
-      const width = Number(this.computedWidth)
+      const width = Number(this.miniVariant ? this.miniVariantWidth : this.width)
```

Comment:
```suggestion
      const width = Number(this.computedWidth)
```

Reply:
this is intended. computedWidth is for the component's width, not the layout area that it effects. Expand on hover creates a larger computed width but we don't want to use that for the layout.

Unless you know something that I don't.

Reply:
Because `computedWidth` is that exact same statement. https://github.com/vuetifyjs/vuetify/blob/ef1fb632c93542d1d70d477cce53a6a2a3d14c6a/packages/vuetify/src/components/VNavigationDrawer/VNavigationDrawer.ts#L170-L172

Reply:
Fuckin hell I'm blind, nvm. Maybe that should be a separate computed property.